From e3d03a826e3aaa33095d6db92d7008791f191c34 Mon Sep 17 00:00:00 2001
From: Rei Oliveira <Reinildes.JoseMateusOliveira@windriver.com>
Date: Wed, 19 Oct 2022 12:54:01 -0300
Subject: [PATCH 3/5] Increase timeout from 90s to 100s

This puppet module is used by the bootstrap manifest to execute
openstack commands. After the change to debian, this 90s timeout
is not enough for some types of hardware.

Signed-off-by: Rei Oliveira <Reinildes.JoseMateusOliveira@windriver.com>

[rebase onto 25.0.0-1: the context is 90s instead of 40s]
Signed-off-by: Michel Thebeau <Michel.Thebeau@windriver.com>
---
 lib/puppet/provider/openstack.rb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/puppet/provider/openstack.rb b/lib/puppet/provider/openstack.rb
index fe6e559..8fffdbb 100644
--- a/lib/puppet/provider/openstack.rb
+++ b/lib/puppet/provider/openstack.rb
@@ -14,7 +14,7 @@ class Puppet::Provider::Openstack < Puppet::Provider
   commands :openstack_command => 'openstack'
 
   @@no_retry_actions = %w(create remove delete)
-  @@command_timeout  = 90
+  @@command_timeout  = 100
   @@request_timeout  = 300
   @@retry_sleep      = 10
   class << self
-- 
2.34.1

